metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_lineage_v2.py

```diff
@@ -240,6 +240,7 @@ def get_known_query_lineage(
 
         downstream_table_urn = self.dataset_urn_builder(dataset_name)
 
+        # TODO: Can KnownQueryLineageInfo support unknown query lineage i.e. query = None
```

Comment:
no it can't, because it tries to generate a fingerprint based on the query

Reply:
removed this TODO